### ISSUE TITLE
Sets of colours missing semantic groupings

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#226]
Colour swatches with images next to them are probably very difficult to understand for screen reader users who cannot see. It's mostly that the visual unit is not as clear programmatically and they are missing some context. Ideally they should be semantically grouped in some way.

For example, consider the following text, which is an approximation of what a screen reader user might hear when reading the bottom of [Colour > Social](https://govuk-brand-guidelines.netlify.app/colour/social/):

> Primary blue, Copy RGB 29 112 184 button, Copy #1D70B8 button
> Primary yellow, Copy RGB 255 221 0 button, Copy #FFDD00 button
> Text "Get your council bill reduced as a student" in white, with "reduced" in black text within a Primary yellow lozenge highlight. Above is a large Primary yellow arrow. Image background is in Primary blue. image

What's missing is the easy to understand context that these three things are related. The context is there because the alt text is mentioning all of those colours that are referenced before. But it's all in the text and not in the semantics and therefore harder to understand.

That is made worse with the companion colour set. Although the colours are technically semantically linked because one set of colours is in one description list, because description lists are not very well supported by all screen readers that distinction can get lost.

This fails WCAG 1.3.1 Info and Relationships.

The easiest way to fix this would be with some (hidden or visible) headings that describe each group. That would also make it easier to navigate to the next group.
Although other solutions might be better. (Among the specialists we thought about adding `hr`s or putting each group into a list item. But those particular examples are not as good as adding headings in this case.)

## Who needs to be involved

Content designer, developer, designer, accessibility specialist

## Who needs to review

Anyone in the squad

**Comment by calvin-lau-sig7** [comment]
Writing headings would be the simplest solution. Headings seem obvious for the tonal examples, but trickier for the companion colour set.

For example:

Tonal colour examples
- Green tones
- Red tones
- Purple tones

Companion colour set
(Naming as colour on colour, using "Primary" or "Accent" as proper names, and describing <50% tints as "light"
- Primary blue and Primary yellow
- Primary red on light magenta
- Primary magenta on light blue
- Purple on Primary yellow
- Purple on Accent blue
- Blue on Accent green
- Accent blue on green
- Accent blue on Primary blue
- Primary yellow on red

Design impact is that this would make this page significantly longer, need to check with designers.

**Comment by calvin-lau-sig7** [comment]
Okay, added into PR #243.

Interestingly, I think the headings look fine in the H4 styling. I tweaked the site structure a little bit to make the sets consistent, which still makes semantic sense (and I know we also could've customised the typeface used for the heading but it was not needed).

I've also reverted to naming the colours specifically, using their tint and percentages, rather than trying to simplify this as "light", etc. as based on work in the other squad, apparently this might be going away soon in the Design System as well.
